deploy/example/noobaa-storagealert.yaml

```diff
@@ -1,14 +1,14 @@
 apiVersion: alerts.monstorak.org/v1alpha1
 kind: StorageAlert
 metadata:
-  name: nooba-storagealert
+  name: noobaa-storagealert
   namespace: storage-monitoring
 spec:
   storage:
-  - provider: "nooba"
+  - provider: "noobaa"
     version: "v1.0"
     namespace: "openshift-storage"
-    serviceMonitor: "nooba-mgr"
+    serviceMonitor: "noobaa-mgr"
   prometheus:
     namespace: "openshift-monitoring"
-    label: "prometheus:k8s"
\ No newline at end of file
+    label: {"prometheus":"K8s"}
```

Comment:
Please check the case sensitivity (if you have intentionally put "K8s", then fine). We can even ignore this as this is just an example.

Reply:
just an example :)